docs/rule-jsdoc.md

```diff
@@ -186,3 +188,56 @@ In the above example, `my-package/foo` is a self reference that connects to `src
 When `treatSelfReferenceAs: external`, this import is always allowed even though `something` is a package-private export because it is treated like an import from an external package.
 
 When `treatSelfReferenceAs: internal`, this import is disallowed because import from `my-package/foo` is treated like an import from `src/somewhere/foo.ts`.
+
+### `excludeSourcePatterns`
+
+_Default value: `[]`_
+
+An array of glob patterns for source paths to exclude from the importability check. When importing from a module that matches one of these patterns, the import-access/jsdoc rule will not apply any restrictions, regardless of the JSDoc annotations or defaultImportability setting.
```

Comment:
It'd be great to explicitly mention where relative paths are resolved from. In this case I believe it is from the project root.